From 9790dd08de71b65598da3b0972b522e4ef6ba5d5 Mon Sep 17 00:00:00 2001
From: Martchus <martchus@gmx.net>
Date: Sun, 25 Sep 2016 21:24:15 +0200
Subject: [PATCH 1/3] Ensure QML dev tools is built as static library

This library must be built as static library even when
using CONFIG+=shared. Otherwise it fails to link.
---
 src/qmldevtools/qmldevtools.pro | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/qmldevtools/qmldevtools.pro b/src/qmldevtools/qmldevtools.pro
index 05aa6f66f..e73c0c23d 100644
--- a/src/qmldevtools/qmldevtools.pro
+++ b/src/qmldevtools/qmldevtools.pro
@@ -2,6 +2,8 @@ option(host_build)
 TARGET     = QtQmlDevTools
 QT         = core-private
 CONFIG    += minimal_syncqt internal_module qmldevtools_build generated_privates
+CONFIG    -= shared
+CONFIG    += static staticlib
 
 MODULE_INCNAME = QtQml
 INCLUDEPATH += $$OUT_PWD/../qml
-- 
2.19.2

